scico/operator/_operator.py

```diff
@@ -47,7 +49,11 @@ def _wrap_mul_div_scalar(func: Callable) -> Callable:
 
     @wraps(func)
     def wrapper(a, b):
-        if np.isscalar(b) or isinstance(b, jax.core.Tracer):
+        if (
+            snp.isscalar(b)
+            or isinstance(b, jax.core.Tracer)
+            or (isinstance(b, jaxlib.xla_extension.ArrayImpl) and b.size == 1)
```

Comment:
I don't think `jax.core` or `jaxlib` should be involved. `(isinstance(b, jax.Array)` should work for arrays and tracers.

Reply:
According to https://jax.readthedocs.io/en/latest/jax.typing.html#jax-typing-best-practices `jax.Array` is currently "annotation for any JAX array or tracer" so `Tracer` is redundant. It probably wasn't when we first implemented this.